packages/browser/src/ItownsUtil.js

```diff
@@ -415,3 +415,37 @@ export function focusC3DTilesLayer(itownsView, layer) {
     horizontalDistance: 200,
   });
 }
+
```

Comment:
I think we don't need these functions. If you want to apply some treatment to tiles with geometry I rather doing something like that:

```js
layer.object3D.traverse((child) => {
 if(!isNaN(child.tileId) && child.tileId >= 0 && child.tileId < layer.tileset.tiles.length ){
  // here child is a tile with geom and tileId
 }
});



```